install.js

```diff
@@ -0,0 +1,8 @@
+// F5 -> npm start -> npm prestart -> this script.
+// Runs `npm install` only if needed. Otherwise skips (and saves time).
+
+const fs = require('fs')
+const { execSync } = require('child_process')
+
+if (!fs.existsSync('node_modules'))
```

Comment:
This can be done in tasks.json, this JS file shouldn't be needed.

Reply:
Couldn't find a way to do it in only `tasks.json`. Would appreciate any tips here.